.github/workflows/go.yml

```diff
@@ -63,3 +63,23 @@ jobs:
         with:
           name: junit.xml
           path: test_reports/junit.xml
+  go-integration-tests:
+      runs-on: buildjet-4vcpu-ubuntu-2204
+      strategy:
+        fail-fast: false
+        matrix:
+          go: [ '1.18' ]
+      steps:
+        - uses: actions/checkout@v3
+        - uses: ./.github/workflows/go-setup
+        - run: make build-ci
+        - run: make tests-e2e-setup
+        - run: make tests-e2e-no-setup
+          env:
+            INTEGRATION_ENABLED: true
+        - run: make junit-report
+        - name: Upload junit report
+          uses: actions/upload-artifact@v2.2.4
```

Comment:
Instead of just uploading an artefact, can we have something that understands junit reports? So that we can have a nice dashboard showing which tests passed and failed, how long they took etc.
Maybe something like this: https://github.com/marketplace/actions/junit-report-action

Reply:
+1 for a dashboard!!

Reply:
This sounds good but I'd vote for a new story on it.  It'd be nice to get this merged in.

Reply:
That sounds useful but I agree it's a separate task and I'd like this merged.

Reply:
Ok. Let's merge it.